starcheck/src/starcheck.pl

```diff
@@ -49,21 +49,43 @@
 use Inline Python => q{
 
 import os
+import six
 from chandra_aca.star_probs import set_acq_model_ms_filter
 import starcheck
 from starcheck.pcad_att_check import make_pcad_attitude_check_report, check_characteristics_date
 from starcheck.calc_ccd_temps import get_ccd_temps
 from starcheck.version import version
 
+def debyte_kwargs(kwargs):
```

Comment:
I could probably use a hand simplifying this @taldcroft .  Looks like when I pass a dictionary from Perl as an argument, all the string keys and values are now bytes, so here I've pretty much just looped over everything to decode.